### ISSUE TITLE
Have an endpoint for switching off Visual Property bypass

### ISSUE THREAD

**Issue by dotasek** [cytoscape/cyREST#6]
In CyREST 3.5, several endpoints can set the bypass property to true, but there is no endpoint that can set it back to false.

**Comment by dotasek** [comment]
The visual property bypass can be set to null, effectively removing the bypass, by using PUT /v1/networks/{networkId}/views/{viewId}/{objectType}/{objectId} with the following message body:

```json
[
  {
    "visualProperty": "NODE_BORDER_PAINT",
    "value": null
  }
]
```
This should be reflected in the docs, or else it will be difficult for users to find/use.

**Comment by dotasek** [comment]
Setting visual property bypass is a hack, and ignores the functionality of something like labels, where the label bypass can value can be set to null to specify no label.

**Comment by dotasek** [comment]
Solved in issues/6 branch. Planning to merge with develop.

**Comment by dotasek** [comment]
Still remains in issues/6 branch. Waiting for 3.6 release before merging to develop.

**Comment by dotasek** [comment]
Merged to master and develop. Awaiting release.

**Comment by dotasek** [comment]
Closed in 3.6.1 release
